linera-service/tests/linera_net_tests.rs

```diff
@@ -1015,7 +1015,7 @@ async fn test_wasm_end_to_end_non_fungible(config: impl LineraNetConfig) -> Resu
     let nft1_minter = account_owner1;
 
     let nft1_blob_bytes = b"nft1_data".to_vec();
-    let nft1_blob_hash = CryptoHash::new(&BlobBytes(nft1_blob_bytes.clone()));
+    let nft1_blob_hash = CryptoHash::new_from_bytes(&nft1_blob_bytes);
```

Comment:
Wouldn't simply `new` work here? `nft1_blob_bytes` are already `Vec<u8>`.

Reply:
I don't think we're implementing `BcsHashable` for `Vec<u8>`, for the above reason.

Reply:
We also want to inject a meaningful seed as domain separator